scripts/tests/mainnet/constant.ts

```diff
@@ -0,0 +1,6 @@
+export const constants = {
+  address_zero: "0x0000000000000000000000000000000000000000",
+  native_address: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+  max_value:
+    "115792089237316195423570985008687907853269984665640564039457584007913129639935",
+};
```

Comment:
Why again in this folder? https://github.com/Instadapp/dsa-connectors/blob/d862b9b5b708dd408f79872ffca0040bf42a78e2/scripts/constant/constant.ts

Reply:
we can remove this from `mainnet/constants`

Reply:
removed